docs/User/SWCI_APIs.md

```diff
@@ -38,7 +38,7 @@ The python3 program needs to import the SWCI class, and then use the below APIs.
 |`listTasks()`|This method displays the list of tasks that are registered into the Swarm Learning network.| |
 |`listTrainingContracts()`|This method displays the list of training contracts registered with Swarm Learning network.| |
```

Comment:
Need to add:

|`getTaskrunnerStatus() <trName:string>`

|Displays the current status of the specified Taskrunner.<br> `trName`: Specifies the unique name of Taskrunner.<br> Provides below information like,<br>- TASK NAME – Current running task or ‘Empty’ if no task is assigned.<br>- PEER TYPE – SL for RUN_SWARM tasks, SWOP for all other tasks.<br>- TASK STATE – Current task state.<br>- ACTIVE, COMPLETED and FAILED PEERS information.<br>- TIME STAMPs of various events on the TASKRUNNER contract.

2) createContext()  needs to reflect the new argument in the corresponding SWCI command